CODEOWNERS

```diff
@@ -411,6 +411,7 @@ build.json @home-assistant/supervisor
 /homeassistant/components/fortios/ @kimfrellsen
 /homeassistant/components/foscam/ @skgsergio
 /tests/components/foscam/ @skgsergio
+/homeassistant/components/frank_energie/ @bajansen @DCSBL
```

Comment:
Hey @bajansen, do you still want to be a codeowner?

Reply:
That would be great. You can put your name first if you want/if it means anything to you.

Reply:
Nah lets keep it alphabetical 👍